src/test/java/technology/semi/weaviate/integration/client/WeaviateTestGenerics.java

```diff
@@ -40,11 +40,13 @@ public void createWeaviateTestSchemaFood(WeaviateClient client) {
             .dataType(Arrays.asList(DataType.STRING))
             .description("name")
             .name("name")
+            .tokenization("field")
```

Comment:
maybe let's put the possible tokenization values into enum?

Reply:
changed to string constants (similarly to DataType)